README.md

```diff
@@ -39,6 +40,16 @@ In this case, you can make use of this extension and pre-spawn `Shoot`s, which a
 Of course, there is a trade-off, as your pre-spawned shoots will consume some resources (either in terms of money, if running in a public cloud, or in terms of physical resources).
 However, in certain scenarios, this approach will dramatically improve the effectiveness of you CI-workflow.
 
+## `shoot-info-envsubst` `ConfigMap`
+
+If the extension is enabled for a shoot cluster a `ConfigMap` named `shoot-info-envsubst` with information about the shoot
+is created in the flux namespace. The `ConfigMap` can be used in [`substituteFrom`](https://fluxcd.io/flux/components/kustomize/kustomizations/#post-build-variable-substitution)
+in `Kustomizations`. The following information is provided:
+
+- SHOOT_INFO_TECHNICAL_ID
+- SHOOT_INFO_NAME
+- SHOOT_INFO_CLUSTER_IDENTITY
```

Comment:
A nice documentation would also describe the meaning of these fields along with an example value 😉

Reply:
I thought the names would be self explanatory for users of gardener. But I can provide a table.